udata_gouvfr/theme/theme.py

```diff
@@ -73,9 +73,13 @@
 
 
 @cache.memoize(50)
-def get_blog_post(url, lang):
+def get_blog_post(lang):
```

Comment:
Removing the parameter from the function's signature will make the `memoize` decorator unaware of multiple calls with different URLs. But we don't call it multiple times :)